### PR TITLE
[DAT-542] - Create "Payment Method" section (readonly view)

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Regarding Junior's comment about create different PRs. This idea is to make the correction easier.

The current PR contains only the form for the readonly view with mocked data and also I created some styles that I will remove when Gus finished with the design in the UI library. In the next PR I will include the integrations with the APIs.

**Changes:**

- Installed the ```react-credit-cards``` package that is is used for the credit card.
- Created the readonly form for the 'Payment method' section.
- Created mocked data used in the view.
- Created the temporary styles used in the view. Regarding it I added ```TODO``` in the code.
- Added UTs for the view.

Screenshot:

image

 **Task:** [DAT-542](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-542)

### CODE REVIEW

**Comment by jcamposmk** [comment]
:+1:

File: src/components/ContactPolicy/ContactPolicy.js [hunk 1]
```diff
@@ -362,7 +362,7 @@ export const ContactPolicy = InjectAppServices(
         actionUrl={_('contact_policy.promotional.upgrade_plan_url')}
         logoUrl={_('common.ui_library_image', { imageUrl: 'icon-pcontacto.svg' })}
         //TODO: Replace icon and image urls
-        previewUrl={('common.ui_library_image', { imageUrl: 'no-list.jpg' })}
+        previewUrl={_('common.ui_library_image', { imageUrl: 'no-list.jpg' })}
```
Comment (cbernat) [inline src/components/ContactPolicy/ContactPolicy.js:365]: ❤️

**Comment by cbernat** [comment]
👍

**Pull request merged**